internal/controller/constants.go

```diff
@@ -26,7 +26,7 @@ const (
 	emulatorModeFalse                = "false"
 	emulatorModeTrue                 = "true"
 	AttributeMediaExtensions         = "me"
-	instaSliceOperatorNamespace      = "default"
+	instaSliceOperatorNamespace      = "instaslice-system"
```

Comment:
Do we need both instasliceOperatorNamespace and operatorDeployNamespace?

Reply:
Good catch, Variable is used in the code, I want to minimize the code change, I agree we should stick to one, maybe we create an issue around it for now?

Reply:
While you are here, I say just drop it.